scripts/devservices-healthcheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This is used for healthchecking because docker running healthchecks
+# keeps the CPU too warm and if a container becomes unhealthy, there's no
+# notification system or anything that would make it more easily visible.
+
+# In the future, more healthchecks should be added to make it more helpful
+# as a troubleshooting script. Right now this is just used by CI.
+
+# TODO: iterate through all sentry_ containers,
+#       and warn on anything that's expected to run but isn't running.
+# TODO: spawn subprocess for each container,
+#       specify distinct retries and other parameters.
+#       would also be nice to have a single ^C work.
+
+try=1
+to=3
+start=0
+delay=5
+
+sleep "$start"
```

Comment:
Do we sleep for 0 seconds? What's the purpose?

Reply:
Say we expect postgres container to take like, 3 seconds from the start to become ready. It'll likely fail healthchecks for those 3 seconds so best not to send them. If you have a large delay/interval then when you fail the initial healthcheck you wait for too long - what happened before was we had start=0 and delay=30, so we'd fail the 1st healthcheck every time then wait 30s.

We could either come up with a better start value or my idea was to just opt out of docker running healthchecks in the background all the time (explained why in comments) and instead take the approach I have here. In the context of CI, we're doing other stuff right now before checking health so postgres is very likely to be ready by the time we start healthcheck. In the context of being a troubleshooting tool, best to have start=0 just to be as fast as possible.